### ISSUE TITLE
Compilation of empty <title>.

### ISSUE THREAD

**Issue by vprunet** [manuel-serrano/hop#65]
Hi,

The following example is not compiled as expected. The missing title is replaced by what should be the body.
I found this issue by copying a third party html example.

Vincent
```js
service foo() {
  return <html>
  <head>
    <meta charset="utf-8">
    <title></title>
  </head>
  <body>
    <div> coucou</div>
  </body>
</html>;
}
```

**Comment by manuel-serrano** [comment]
Hi Vincent,

Which branch are you using? With 3.2.0 it seems correct. I will check with 3.1.0 (it is currently being compiled :-)

**Comment by manuel-serrano** [comment]
Forget about my last message. I have understood what's going wrong. Hop generates <title/>, which is why I though the generated code was correct. This is apparently not a well formed HTML fragment and then browsers interpret this form as if the following was indeed the body of the title. I will fix this right away...

**Comment by manuel-serrano** [comment]
I have fixed the problem and I have pushed a new 3.1.x version. I will push the patch for 3.2.x no later than tomorrow. 

Thanks again for the report.

**Comment by vprunet** [comment]
Thank you.
